common/components/utils/cx.js

```diff
@@ -0,0 +1,11 @@
+class cx {
```

Comment:
So is this class mainly for saving typing?

Reply:
yepp